course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -224,7 +224,10 @@ def get_expected_data(cls, course, request):
     def setUp(self):
         super().setUp()
         self.request = make_request()
+        self.edx_org_short_name = 'edX'
         self.course = CourseFactory(partner=self.partner)
+        # Course of edx organization
+        self.course_2 = CourseFactory(partner=self.partner, authoring_organizations__key=self.edx_org_short_name)
```

Comment:
move these changes to test case `test_filter_courses_on_edx_org_short_name` since this has no impact on existing test cases.

Reply:
Done

Reply:
this code is still here.